src/api/app/policies/appeal_policy.rb

```diff
@@ -0,0 +1,16 @@
+class AppealPolicy < ApplicationPolicy
+  def new?
+    create?
+  end
+
+  def show?
+    return false unless Flipper.enabled?(:content_moderation, user)
+    return true if record.appellant == user
+
+    user.is_admin? || user.is_moderator? || user.is_staff?
+  end
+
+  def create?
+    DecisionPolicy.new(user, record.decision).appeal?
```

Comment:
I wouldn't delegate this to `DecisionPolicy` which then delegate to another policy depending on the reportable. We can trust that if a user reported something, it's because the user had the right to create the report. This authorization check was done at that point in time. So in this case, the user can appeal a decision which cleared the report.

Reply:
@krauselukas and I talked. We pushed some changes.